org.gtk.Gtk3theme.Mojave-light.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Note we don't display themes on https://flathub.org so usually appdata is more barebone like https://github.com/flathub/org.gtk.Gtk3theme.Pop/blob/master/org.gtk.Gtk3theme.Pop.appdata.xml